book_source/03_intermediate_users_guide/01_pecan_xml.Rmd

```diff
@@ -1,195 +1,487 @@
+* `threshold`: threshold for Gelman-Rubin convergence diagnostic (MGPRF); default is 1.2.
 
-## Brown Dog Configuration
+This information is currently used by the following PEcAn workflow function:
 
-This section describes how to connect to [Brown Dog](http://browndog.ncsa.illinois.edu). This will allow for conversions of data (currently only met data).
+- `PEcAn.MA::run.meta.analysis` - ???
 
-#### functions: 
+### `model`: Model configuration {#xml-model}
 
-`met.process()`
+This section describes which model PEcAn should run and some instructions for how to run it.
 
 ```xml
-<browndog>
-	<url>...</url>
-	<username>...</username>
-	<password>...</password>
-</browndog>
+<model>
+	<id>7</id>
+	<type>ED2</type>
+	<binary>/usr/local/bin/ed2.r82</binary>
+	<job.sh>module load hdf5</job.sh>
+	<config.header>
+		<!--...xml code passed directly to config file...-->
+	</config.header>
+</model>
 ```
 
-* **url** : [required] endpoint for Brown Dog to be used.
-* **username** : [optional] username to be used with the endpoint for Brown Dog.
-* **password** : [optional] password to be used with the endpoint for Brown Dog.
+Some important tags are as follows:
 
+- `id` -- The unique numeric ID of the model in the Bety `models` table. If this is present, then `type` and `binary` are optional since they can be determined from Bety.
+- `type` -- The model "type", matching the Bety `modeltypes` table `name` column. This also refers to which PEcAn model-specific package will be used. In PEcAn, a "model" refers to a specific version (e.g. release, git commit) of a specific model, and "model type" is used to link different releases of the same model. Model "types" also have specific PFT definitions and other requirements associated with them (e.g. the ED2 model "type" requires a global land cover database).
+- `binary` -- The file path to the model executable. If omitted, PEcAn will use whatever path is registered in Bety for the current machine. 
+- `job.sh` -- Additional options added to the `job.sh` script, which is used to execute the model. This is useful for setting specific environment variables, load modules, etc.
 
-## PFT Selection
+This information is currently used by the following PEcAn workflow function:
 
-The PEcAn system requires at least 1 PFT (Plant Functional Type) to be specified inside the `<pfts>` section. 
+`write.configs()`, `run.models()`
 
-#### functions:  
+#### Model-specific configuration {#xml-model-specific}
 
-`get.traits()`
+See the following:
 
+* [ED2][ED2 Configuration]
+* [SIPNET][SIPNET Configuration]
+* [BIOCRO][BioCro Configuration]
 
-#### tags
+#### ED2 specific tags {#xml-ed}
```

Comment:
Only documenting ED's special tags here is a bit odd and hard to justify. Seems like we should have all model-specific tags linked to model specific sections. The list above (ED2, SIPNET, BIOCRO) is clearly very very out of date -- all models that have model-specific tags should be there (LINKAGES, FATES, JULES, etc), and models that don't have special tags should not (e.g. I'm unaware of SIPNET actually having extra configurations). Alternatively, if that's hard to maintain, there could just be one link to the overall model documentation so people can look up individual models.